### ISSUE TITLE
Markdown tables - change from v2.0 - deliberate?

### ISSUE THREAD

**Issue by ddrury** [fisharebest/webtrees#4426]
Previously markdown tables css had `border-collapse: collapse;` now it is over-ridden in the `<theme>.css` and set to to `border-collapse: separate;`

**Comment by ddrury** [comment]
The markdown class is not applied anywhere in webtrees currently, lack of this css is the cause of the problem (although I think only tables are affected)

This is also the cause of https://github.com/fisharebest/webtrees/issues/4441

**Comment by fisharebest** [comment]
> deliberate?

No.  We cannot add `<div class="markdown"></div>" in the markdown renderer, as it forces the text to be a block on the next line.  See recent bug reports.

So, we must now add it wherever it is used, and not for single-line text.

I guess I must have missed some of the cases.

**Comment by ddrury** [comment]
Only markdown tables are missing some styling which seems to me to be consistent for whatever device it is viewed on. Therefore the simplest solution for me is to add class markdown to `const MarkdownFactory::CONFIG_MARKDOWN table->attributes->class`
and in _markdown.css delete everything not related to tables.

BTW if you wish to keep `.markdown th {...}` you need to add !important as its being over-ridden by `<style>.css .wt-facts-table`

**Comment by fisharebest** [comment]
> No. We cannot add `<div class="markdown"></div>` in the markdown renderer, as it forces the text to be a block on the next line. See recent bug reports.

I had forgotten about this - and recently added this `<div>`.

I need to investigate why we removed it initially...
